### DISCUSSION TITLE
[Feature] Table of Contents Markdown Support

### DISCUSSION THREAD

**Discussion by Steven-Harris** [verdaccio/verdaccio#1298] [Ideas]
The idea is to support relative link via fragments like on some [npmjs repos](https://www.npmjs.com/package/@angular/cli#installation) - this would improve usability of the README page but really it's just a "nice-to-have". 

```
# Table of Contents
1. [Example](#example)
2. [Example2](#example2)
3. [Third Example](#third-example)

## Example
## Example2
## Third Example
```
Clicking on "Example2" would auto scroll to the "Example2" section of the page.

**Comment by juanpicado** [comment]
Something like GitHub and npmjs does, yeah, I think is handy. Do you know what library they use for it?

**Comment by Steven-Harris** [comment]
It might be this one [https://github.com/github/markup](https://github.com/github/markup)

**Comment by Steven-Harris** [comment]
Oh that is just the library to determine what library to use. Looks like they use [https://github.com/gjtorikian/commonmarker](https://github.com/gjtorikian/commonmarker) for markdown files. It's a ruby library so it might not be the best for integration with verdaccio

**Comment by ghost** [comment]
I use mod-markdown as an Apache mod which I think uses Discount markdown engine, which is in C, which in turn is a port of a python markdown lib, I cant recall then name.  But yes, this feature would be great!
